code/controllers/subsystem/movement/movement_types.dm

```diff
@@ -797,6 +822,10 @@
 		return
 	potential_directions = directions
 
+/datum/move_loop/move_rand/compare_loops(datum/move_loop/loop_type, priority, flags, extra_info, delay, timeout, list/directions)
+	if(..() && potential_directions == directions) //i guess this could be usefull if actually it really has yet to move
+		return TRUE
```

Comment:
A: usefull
B: This check will fail when you want it to succeed
```

Reply:
right potential_directions is a list whops

Reply:
> 

whoa this actually did not load for me first thats really useful